rotkehlchen/chain/evm/decoding/aave/common.py

```diff
@@ -329,6 +336,51 @@ def _decode_repay(
 
         return return_event, repay_event
 
+    def _maybe_decode_mint_events(self, context: DecoderContext) -> None:
+        """Decode possible mint events in transactions. Those are result of the aW(NATIVE)
+        contract minting and transfering before withdrawing the native asset. Code at
+        https://scrollscan.com/address/0xFF75A4B698E3Ec95E608ac0f22A03B8368E05F5D#code
+
+        Example tx: https://scrollscan.com/tx/0x65cd06fd54a10052c3d9084d14d28c06e2bb328b1ec39730fab9284cb529d068
+        """
+        mint = None
+        for tx_log in context.all_logs:
+            if tx_log.log_index >= context.tx_log.log_index:
+                break  # it is possible to not have mint events. Also don't query after the pool event  # noqa: E501
+
+            if tx_log.topics[0] == MINT:
+                token = EvmToken(evm_address_to_identifier(
+                    address=tx_log.address,
+                    token_type=EvmTokenKind.ERC20,
+                    chain_id=self.evm_inquirer.chain_id,
+                ))
+                mint = (
+                    hex_or_bytes_to_address(tx_log.topics[2]),  # onBehalfOf
+                    asset_normalized_value(  # amount minted
+                        amount=hex_or_bytes_to_int(tx_log.data[0:32]),
+                        asset=token,
+                    ),
+                    token,
+                )
+                break
```

Comment:
always remember the newline after break. Absolutely essential in python. Otherwise super hard to read. Also told you in last review.

```suggestion
                break

```